example/README.md

```diff
@@ -0,0 +1,52 @@
+# Building examples
+
+Vecty fully supports all three major Go <-> Web compilers thanks to its minimal dependencies:
+
+- Go 1.12+ WebAssembly support
+- [GopherJS](https://github.com/gopherjs/gopherjs) (Go to JavaScript transpiler)
+- [TinyGo](https://github.com/tinygo-org/tinygo) WebAssembly support (produces tiny binaries, but with tradeoffs)
```

Comment:
:grimacing: is this right? TinyGo doesn't support all of `reflect` yet. Have you tested it with TinyGo?

Reply:
Yeah, I haven't updated this file yet & TinyGo most likely doesn't work. Will fix.